SecondStage/module28/src/test/java/page/googlecloud/GoogleCloudEmailYourEstimatePage.java

```diff
@@ -0,0 +1,69 @@
+package page.googlecloud;
+
+import page.CommonAbstractPage;
+import page.minuteinbox.MinuteInboxHomePage;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.ArrayList;
+
+public class GoogleCloudEmailYourEstimatePage extends CommonAbstractPage {
+
+    // Field to store copied temporary email address.
+    private String emailAddress;
```

Comment:
Page objects should contain only data relative to the page it is representing. For business objects, we may introduce a separate layer